scripts/install/RegisterItemCommentElasticsearchAdapter.php

```
@@ -0,0 +1,54 @@
+namespace oat\taoAdvancedSearch\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\taoAdvancedSearch\model\Comment\ElasticsearchItemCommentAdapter;
+use oat\taoItems\model\Comment\ItemCommentPersistenceProxy;
+
+/**
+ * Replaces taoItems RDF comment persistence with Elasticsearch when AS is installed.
+ */
+class RegisterItemCommentElasticsearchAdapter extends InstallAction
+{
+    public function __invoke($params = [])
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)